src/components/card/createCards.js

```diff
@@ -188,4 +188,41 @@ function crtCardBtn() {
     cardBtn.textContent = '+';
 
     return cardBtn;
+}
+
+function addDish(event) {
+
+    if(event.target.classList.contains('card__btn')) {
+        const cardId = +event.currentTarget.id;
+
+        try {
+            getCardDatabById(cardId)
+            .then(dataOrder => {
+
+                const dataCard = getOrdersDataToStorage('Orders');
+
+                if(dataCard === null) {
+                    const arrForDataLocalStorage = [];
+                    arrForDataLocalStorage.push(dataOrder);
```

Comment:
you can reduce this block
 
 if(dataCard === null) {
                    setOrdersDataToStorage([dataOrder]);
                    }